spec/features/user_sees_sleeplog_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+feature 'user signs in', %{
+  As an authenticated user
+  I want to be able to see my sleep information
+  So that I can track changes in my sleep behavior
+} do
+
+# Acceptance Criteria:
+# [X] I can navigate to my sleep log from any page
+# [ ] I can choose to see a display of sleep gotten over the last week, month, or year
+# [ ] I can see my mood from the night before (based on the diary log), to see if my mood affected my sleep
```

Comment:
Line is too long. [107/80]<br>Incorrect indentation detected (column 0 instead of 2).